internal/app/cmd/cmd.go

```diff
@@ -87,6 +90,12 @@ func runHTTP(cmd *cobra.Command, cfg killgrave.Config) error {
 	done := make(chan os.Signal, 1)
 	defer close(done)
 
+	logLevel, err := log.ParseLevel(cfg.Log.Level)
+	if err != nil {
+		return fmt.Errorf("Could not parse log level: %v", err)
```

Comment:
I don't think we want to print the `logrus` error here. Have you tried to reproduce this scenario and see how the message looks like? 🤔

Reply:
Btw, wdyt about moving these lines inside `configureLogging`, so we can type `killgrave.Config.Log.Level` with the specific type, and make sure that when we call `runHTTP`, the config is valid?

Reply:
yeah, it doesn't make sense that `configureLogging` doesn't fail on an invalid log config, moving it

Reply:
moved! left only the `log.SetLevel` call in `runHTTP`